docs/cli/run.md

```diff
@@ -11,61 +11,48 @@ slug: "run"
 # `cortex run`
 :::info
 This CLI command calls the following API endpoint:
-- [Download Model](/api-reference#tag/models/post/v1/models/{modelId}/pull) (The command only calls this endpoint if the specified model is not already downloaded.)
+- [Download Model](/api-reference#tag/models/post/v1/models/{modelId}/pull) (The command only calls this endpoint if the specified model is not downloaded yet.)
+- Download Engine (The command only calls this endpoint if the specified engine is not downloaded yet.)
 - [Start Model](/api-reference#tag/models/post/v1/models/{modelId}/start)
 - [Chat Completions](/api-reference#tag/inference/post/v1/chat/completions) (The command makes a call to this endpoint if the `-c` option is used.)
 :::
 
 This command facilitates the initiation of an interactive chat shell with a specified machine-learning model.
 
-
-This command supports both local and remote models.
-
 ## Usage
 
 ```bash
-cortex run [options] [model_id]
+cortex run [options] <model_id>
 # With a specified engine
-cortex run [options] [model_id]:[engine]
+cortex run [options] <model_id>:[engine]
 # Start chatting once the model started
-cortex run [options] [model_id]:[engine] -c
+cortex run [options] <model_id>:[engine] -c
```

Comment:
we don't have `-c` for now. It will automatically start to chat when execute `run` command